tests/test_model_methods.py

```diff
@@ -121,6 +123,8 @@ async def test_save_partial(self):
         n_mdl = await self.cls.get(id=self.mdl.id)
         self.assertEqual(n_mdl.name, "Test")
         self.assertEqual(n_mdl.desc, "Something")
+        with pytest.raises(OperationalError):
+            await self.mdl.save(update_fields=["id"])
```

Comment:
Can you make this check a separate test so it is documented behavior? I would also suggest checking the exception message, e.g.`with pytest.raise(OperationalError, match="...")`